src/services/helper/entity-type.js

```diff
@@ -0,0 +1,152 @@
+// DependenciesI
+const httpStatusCode = require('@generics/http-status')
+const common = require('@constants/common')
+const entityTypeQueries = require('../../database/queries/entityType')
+const { UniqueConstraintError } = require('sequelize')
+
+module.exports = class EntityHelper {
+	/**
+	 * Create entity.
+	 * @method
+	 * @name create
+	 * @param {Object} bodyData - entity body data.
+	 * @param {String} id -  id.
+	 * @returns {JSON} - Entity created response.
+	 */
+
+	static async create(bodyData, id) {
+		bodyData.created_by = '0' || id
```

Comment:
Wouldn't this always assign '0' to created_by? 

Considering non-zero string being truthy.

Reply:
The temporary solution has been removed.